README.md

```diff
@@ -1,54 +1,106 @@
-xblock-group-project-v2
-=======================
+# xblock-group-project-v2
+
 
 This XBLock is experimental reimplementation of [Group Project XBlock](https://github.com/edx-solutions/xblock-group-project).
 
 It does *not* work properly in LMS yet.
 
-Configuration variables
------------------------
+## Features
 
-This block uses following configuration variables in the ``XBLOCK_SETTINGS``:
+* Structure - group project is organized around activities and stages. 
+    * Activities are the larger chunks, used to distinguish between logically bound parts of group project. Activities 
+      can contain one or more stages. Grading happens at activity level, i.e. one grade per activity.
+    * Stages are smaller building blocks, representing individual steps towards project completion.
+* Collaborative learning - students are organized into workgroups (aka cohorts) to work on a set of problems in a
+  collaborative way.
+* Team communication - group project interface allows sending emails to individual teammates or entire workgroup.
```

Comment:
@itsjeyd no, "workgroup" - there aren't any mechanism to send message to other group. :)